Formula/qscintilla2.rb

```diff
@@ -18,8 +19,10 @@ class Qscintilla2 < Formula
 
   if build.with? "python3"
     depends_on "pyqt" => "with-python3"
-  else
+  elsif build.with? "python"
```

Comment:
Per my comment in the main thread this should be accompanied by a `option "without-python"` so the default build keeps the Python element.